dali/pipeline/operators/crop/crop.cc

```diff
@@ -29,20 +37,20 @@ DALI_SCHEMA(Crop)
     .AddOptionalArg("crop_pos_y",
                     R"code(Vertical position of the crop in image coordinates (0.0 - 1.0))code",
                     0.5f, true)
-    .AddOptionalArg("image_type",
-                    R"code(The color space of input and output image)code", DALI_RGB, false)
     .AddArg("crop",
             R"code(Size of the cropped image. If only a single value `c` is provided,
  the resulting crop will be square with size `(c,c)`)code", DALI_INT_VEC)
+    .AddParent("CastPermute")
     .EnforceInputLayout(DALI_NHWC);
 
 
 template<>
-Crop<CPUBackend>::Crop(const OpSpec &spec) : Operator<CPUBackend>(spec), CropAttr(spec) {
+Crop<CPUBackend>::Crop(const OpSpec &spec, bool defaultCastPermut) :
```

Comment:
Crop in itself is a useful operator. Handling this through a binary flag doesn't seem like an improvement